.ci/azure-pipelines/build-windows.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         displayName: 'Free Up Space'
       - script: |
           mkdir %BUILD_DIR% && cd %BUILD_DIR%
-          cmake $(Build.SourcesDirectory) -G"%GENERATOR%" -DCMAKE_TOOLCHAIN_FILE=%VCPKG_ROOT%\scripts\buildsystems\vcpkg.cmake -DVCPKG_APPLOCAL_DEPS=ON -DPCL_BUILD_WITH_BOOST_DYNAMIC_LINKING_WIN32=ON -DPCL_BUILD_WITH_FLANN_DYNAMIC_LINKING_WIN32=ON -DPCL_BUILD_WITH_QHULL_DYNAMIC_LINKING_WIN32=ON -DBUILD_global_tests=ON -DBUILD_tools=OFF -DBUILD_surface_on_nurbs=ON
+          cmake $(Build.SourcesDirectory) -G"%GENERATOR%" -DCMAKE_BUILD_TYPE="Release" -DCMAKE_TOOLCHAIN_FILE=%VCPKG_ROOT%\scripts\buildsystems\vcpkg.cmake -DVCPKG_APPLOCAL_DEPS=ON -DPCL_BUILD_WITH_BOOST_DYNAMIC_LINKING_WIN32=ON -DPCL_BUILD_WITH_FLANN_DYNAMIC_LINKING_WIN32=ON -DPCL_BUILD_WITH_QHULL_DYNAMIC_LINKING_WIN32=ON -DBUILD_global_tests=ON -DBUILD_tools=OFF -DBUILD_surface_on_nurbs=ON
         displayName: 'CMake Configuration'
       - script: cd %BUILD_DIR% && cmake --build . --config %CONFIGURATION%
```

Comment:
Does MSVC automatically perform parallelism? If so, we might want to increase the jobs (from 2 to what MSVC uses) on other systems, and if no, add jobs.

Reply:
I have no idea if it does parallelize automatically. Will wait for feedback if someone does know about this.

Reply:
This section of CMakeLists (supposedly) takes care of parallelizm for MSVC:

https://github.com/PointCloudLibrary/pcl/blob/2133857dd8ddd0e80602fc1602651a5afb10f934/CMakeLists.txt#L166-L187